### PR TITLE
[workflow] Test against multiple python versions

### PR DESCRIPTION
**Pull request by archibate**
Related issue = as I promised in #1023 

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

File: .github/workflows/persubmit.yml [hunk 1]
```diff
@@ -7,12 +7,21 @@ jobs:
...
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip ci') || contains(github.event.head_commit.message, '[skip ci]' }}
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
+        #with_opengl: [0, 1]
+        #exclude:
+        #  - python: 3.6
+        #    with_opengl: ubuntu-16.04
+        #  - python: 3.8
+        #    with_opengl: ubuntu-16.04
```
Comment (yuanming-hu) [inline .github/workflows/persubmit.yml]:
```suggestion
        #with_opengl: [0, 1]
        #exclude:
        #  - python: 3.6
        #    with_opengl: ubuntu-16.04
        #  - python: 3.8
        #    with_opengl: ubuntu-16.04
```

Any reason to keep these comments?

Comment (archibate) [reply .github/workflows/persubmit.yml]: Wanna implement this for testing both gpu and cpu, fine, let's do it iapr.

**Comment by archibate** [comment]
Would you like to also test on ubuntu-16.04? I can do it itpr.

File: .github/workflows/persubmit.yml (hunk 1 above)
Comment (yuanming-hu) [reply .github/workflows/persubmit.yml]: Oh, that would be nice! The 6 matrices will run in parallel, right?

**Comment by yuanming-hu** [comment]
> Would you like to also test on ubuntu-16.04? I can do it itpr.

Oh, that would be nice! The 6 matrices will run in parallel, right?

**Comment by archibate** [comment]
> > Would you like to also test on ubuntu-16.04? I can do it itpr.
> 
> Oh, that would be nice! The 6 matrices will run in parallel, right?

Yes, it will, but max to 5 for free users. I would suggest only add a ub16+py37 so we get 4 for CI, and 1 for check-code-format.

**Pull request merged**